types/vcav1/vcatypes.go

```diff
@@ -0,0 +1,136 @@
+package vcatypes
```

Comment:
Refactor this package with the api version you've built this against, I guess it's either 5.7 or 5.11 so create a `types/v57` or `types/v511` dir and put types in there.